tests/test_optislang/test_optislang_integration.py

```diff
@@ -0,0 +1,90 @@
+from tkinter import END
+import pytest
+import os
+import ansys.fluent.core as pyfluent
+from ansys.fluent.core import examples
+from pathlib import Path
+
+@pytest.mark.optislang
+@pytest.mark.integration
+
+def test_simple_solve(load_mixing_elbow_param_case_dat):
+    
+    """
+    This optiSLang integration test performs these steps
+
+    - Reads a case file with and without data file
+    - Gets input and output parameters and creates dictionary
+    - Sets a variation on input parameters
+    - Solve
+    - Reread data
+
+    This test queries the following using PyTest:
+    - Input parameters
+    - Output parameters
+    """
+
+    # Step 1: Setup logging 
+    pyfluent.set_log_level('ERROR')
+
+    # Step 2: Launch fluent session and read case file with and without data file
+    session = load_mixing_elbow_param_case_dat
+    session.check_health() == 'SERVING' 
```

Comment:
This line will actually have no effect as it stands.
```suggestion
    assert session.check_health() == 'SERVING' 
```